src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForBackupDatabaseNow.cs

```diff
@@ -15,29 +22,74 @@ public ShardedOngoingTasksHandlerProcessorForBackupDatabaseNow([NotNull] Sharded
         {
         }
 
-        protected override ValueTask<bool> ScheduleBackupOperationAsync(long taskId, bool isFullBackup, long operationId, DateTime? _)
+        protected override async ValueTask<(long OperationId, bool IsResponsibleNode)> ScheduleBackupOperationAsync(long taskId, bool isFullBackup, long operationId,  DateTime? __)
         {
             var token = RequestHandler.CreateTimeLimitedBackgroundOperationToken();
-
+            
+            // backup might be already running on one of the shards, get the current operation id from that shard if so
+            var shardsOperationId =
+                await RequestHandler.ShardExecutor.ExecuteParallelForAllAsync(new ShardedGetOperationIdForBackupOperation(RequestHandler.HttpContext, taskId));
+            
+            if (shardsOperationId != Constants.Operations.InvalidOperationId)
+                throw new BackupAlreadyRunningException($"Can't start backup with taskId {taskId} because it is already running under operation id {shardsOperationId}")
+                {
+                    OperationId = shardsOperationId,
+                    NodeTag = ServerStore.NodeTag
```

Comment:
This won't always be usable to the client because there might be an instance where the backups started running independently on the shards due to timer. Which means we haven't gone through the orchestrator -> there has been no operation added to the orch, it doesn't know about the backups. If we try waiting on this nodeTag + operationId combo we won't find this operation Id there.